Standards/scs-0101-w1-entropy-implementation-testing.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "SCS Entropy: Implementation and Testing Notes"
+type: Supplement
+status: Draft
+supplements:
+  - scs-0101-v1-entropy.md
+---
+
+## Implementation
+
+We presume that almost nothing has to be done (or indeed can be done), as
+long as the CPUs and VM images are reasonably recent; only the flavor and
+image attributes have to be set:
```

Comment:
This confuses me and misses any intro about what you are trying to discuss. Do you mean Prerequisites or Requirements?

Reply:
The section `Implementation` shows you what you have to do to make your environment compliant with the standard in question. In this case, nothing much can be done, provided that the CPUs and VM images are reasonably recent.